src/adzerk/boot_reload.clj

```diff
@@ -23,11 +23,11 @@
          (map tmp-path)
          (remove static-files))))
 
-(defn- start-server [pod {:keys [ip port ws-host secure?] :as opts}]
+(defn- start-server [pod {:keys [ip port ws-host ws-port secure?] :as opts}]
   (let [{:keys [ip port]} (pod/with-call-in pod (adzerk.boot-reload.server/start ~opts))
         host              (cond ws-host ws-host (= ip "0.0.0.0") "localhost" :else ip)
         proto             (if secure? "wss" "ws")]
-    (util/with-let [url (format "%s://%s:%d" proto host port)]
+    (util/with-let [url (format "%s://%s:%d" proto host (or ws-port port))]
```

Comment:
This will return a url with the correct port but print message with wrong port. The server will be started at `port`.

I'll merge this and fix myself.